src/middlewared/middlewared/schema.py

```diff
@@ -174,6 +174,7 @@ class IPAddr(Str):
 
     def validate(self, value):
         if value:
+            value = value.rsplit('/')[0]  # May have CIDR
```

Comment:
Why don't we validate the rest of the string?

Reply:
I see, you are validating it in the `clean` method. We can create reusable component (e.g. `IPAddr(cidr=True)` for that case